study-builder/tenants/cmi/tenant.yaml

```diff
@@ -54,6 +67,33 @@ clients:
     ]
     client_metadata:
       study: ANGIO
+    jwt_configuration:
+      alg: RS256
+      lifetime_in_seconds: 36000
+    oidc_conformant: true
+  - name: angio-admin-client
+    app_type: spa
+    callbacks: [
+      '##ANGIO_BASE_URL##/admin-login-landing'
+      ##LOCALHOST_ADMIN_CALLBACKS##
+    ]
+    allowed_logout_urls: [
+      '##ANGIO_BASE_URL##',
+      '##ANGIO_BASE_URL##/error',
+      '##ANGIO_BASE_URL##/admin-session-expired'
+      ##LOCALHOST_ADMIN_LOGOUTS##
+    ]
+    web_origins: [
+      '##ANGIO_BASE_URL##'
+      ##LOCALHOST_ORIGINS##
+    ]
+    client_metadata:
+      study: ANGIO
+      isAdminClient: 'true'
```

Comment:
I think it's possible to have a single admin client for all CMI studies. But each client has metadata about which study it is for, so I just created separate clients for each study.